lib/ama/styles/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module AMA
   module Styles
-    VERSION = '1.17.2'
+    VERSION = '1.17.3'
```

Comment:
for the version numbers, the way it works is the first number is a major update, the second is a minor and the third is for bug fixes. So in this instance you'd want to update the minor number rather than the number for bugs.